### DISCUSSION TITLE
Dealer / Reseller Pricing

### DISCUSSION THREAD

**Discussion by cmdigitaluk** [openstore-ecommerce/OpenStore#186] [Q&A]
How do I set up Dealer / Reseller pricing?  I've added a dealer price in the back office and the documentation says "this price is only displayed to users with a "reseller" status on the site (to be defined when setting it up)" But I can't find anymore information on this.  I seem to remember it was done on DNN role but can't remember what other steps I need to take.

We are using an old version of NBright Store version 3.6.7.

As a side question, is the old forum archived anywhere as I would frequently find answers there?

**Comment by DNNMonster** [comment]
First of all time to upgrade...  Lots of improvements beckon to you.

Do you have your buyer in the "Dealer" role?

OS will use a default role of "Dealer" or you can specify something else in the Back Office.

https://github.com/openstore-ecommerce/OpenStore/blob/ce09324695cc817ceb49919fa0dfc902950595ea/Components/StoreSettings.cs#L106

You'll want to double check your that your DNN user is in the "Dealer" role.

And you'll want to check to make sure that your OS BO Admin configuration for the dealer role is proper.

You should see a Store Configuration section in your Back Office and it will contain something like below.
![AdminConfigDealerRole](https://user-images.githubusercontent.com/12777553/130623207-e10a7886-de0d-46b7-a1ca-199ea1295e0e.png)

I'm not sure about the forum.

**Reply by cmdigitaluk** [reply]
Thanks for your help.  Completely agree that we need to upgrade, we will have that discussion with the client.

We discovered that the dealer price was working but you can only see the dealer price when you add the product to the basket. So we modified the product template to display the dealer price when someone has the dealer role set in BO Admin.  

It all works great now.
